libsrc/apple2/prodos_datetime_to_posix_tm.s

```diff
@@ -0,0 +1,99 @@
+;
+; Oliver Schmidt, 14.08.2018
+; Colin Leroy-Mira, 2023 <colin@colino.net>
+;
+; int __fastcall__ prodos_datetime_to_posix_tm(struct tm *tm, struct prodos_datetime *prodos_dt)
```

Comment:
I think it's preferable to have this function return a value to the "internal" struct tm, just like localtime() and gmtime() do.